tools/runtests.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
This file should not be necessary with `busted`?

Reply:
Removed.